as2/6.3/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,113 @@
+= Upgrading and Migrating AS2 Connector to Version 6.x - Mule 4
+:page-aliases: connectors::as2/as2-connector-upgrade-migrate.adoc
+
+Upgrade Anypoint Connector for AS2 (AS2 Connector) from version 5.x to 6.x to take advantage of new features and functionality.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|5.x |6.x
+|===
+
+== Changes
+
+=== 6.3
+
+AS2 Connector 6.3 contains the following changes:
+
+* You can now configure a single AS2 listener endpoint to receive messages from multiple partners.
++
+To implement this feature, a new tab named *Partners* was added to the *As 2 listener* and *As 2 mdn listener* global elements. This tab contains fields that were were moved from the *Partner Config* section of the *Geneneral* tab, as well as some new fields.
```

Comment:
```suggestion
To implement this feature, a new tab named *Partners* was added to the *As2 listener* and *As2 mdn listener* global elements. This tab contains fields that were moved from the *Partner Config* section of the *General* tab, as well as some new fields.
```

Reply:
Sadly, "AS 2 mdn listener" is the name in the GUI.